lib/charms/kubernetes_charm_libraries/v0/multus.py

```diff
@@ -631,11 +666,3 @@ def _on_remove(self, event: RemoveEvent) -> None:
     def delete_pod(self) -> None:
         """Delete the pod."""
         self.kubernetes.delete_pod(self._pod)
-
-    def get_network_attachment_definitions(self) -> list[NetworkAttachmentDefinition]:
```

Comment:
Why are we removing this function?

Reply:
This function is not called anywhere so removing.

Reply:
The charm does not need to call it anymore.

Reply:
it's public, isn't there any chance some charm that uses the lib uses this function too?

Reply:
This is added by me in this PR https://github.com/canonical/kubernetes-charm-libraries/pull/12. That time we were planning to restart pod through the charm. But we decided to change our design and library becomes responsible to decide a pod restart is required or not without an intervention by the charm. This is the reason we do not need this public method anymore.

Reply:
I will remove it in another PR. You are right, this PR do not need to touch this method at the moment.